src/domain/common/lifecycle/lifecycle.dto.transition.ts

```diff
@@ -0,0 +1,14 @@
+import { InputType, Field } from '@nestjs/graphql';
+import { IsOptional, MaxLength } from 'class-validator';
+import { SMALL_TEXT_LENGTH } from '@src/common/constants';
+
+@InputType()
+export class LifecycleEventInput {
```

Comment:
lifecycle.dto.transition --> LifecycleEventInput - was that intentional?

Reply:
Good catch